scripts/functions.R

```diff
@@ -67,11 +67,12 @@ fpr_table_moti <- function(dat = tab_moti_phase2,
                            xref_table = xref_moti_climate_names,
                            site = my_site,
                            ...){
+  df <- dat %>% filter(pscis_crossing_id == site)
   # build left side of table
   tab_results_left <- xref_table %>%
     filter(id_side == 1)
 
-  tab_pull_left <- dat %>%
+  tab_pull_left <- df %>%
```

Comment:
`dat` is a preferred name because `df` means degrees of freedom and has function in other packages. likely not am issue for us but I stopped using it in new funcitons for that reason